Auth0/WebAuthError.swift

```diff
@@ -63,6 +65,11 @@ public enum WebAuthError: CustomNSError {
                 NSLocalizedDescriptionKey: message,
                 WebAuthError.infoKey: self
             ]
+        case .noNonceProvided:
+            return [
+                NSLocalizedDescriptionKey: "You must provide a nonce value for the id_token response type",
```

Comment:
`A nonce value must be supplied when response_type includes id_token in order to prevent replay attacks`